5calls/app/src/main/java/org/a5calls/android/a5calls/model/DatabaseHelper.java

```diff
@@ -124,6 +136,10 @@ public void onUpgrade(SQLiteDatabase db, int oldVersion, int newVersion) {
                     new String[]{Outcome.Status.VM.toString()});
             currentDbVersion = 3;
         }
+
+        if (oldVersion < 4 && currentDbVersion < newVersion) {
+            db.execSQL(STARRED_ISSUES_TABLE_CREATE);
```

Comment:
set `currentDbVersion = 4` in case we upgrade to 5 later. Otherwise we won't go into the next `if`.